library/src/scripts/theming/AddTheme.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ *
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import React from "react";
+import { PlusIcon } from "@library/icons/common";
+import { currentThemeClasses } from "./currentThemeStyles";
+import SmartLink from "@library/routing/links/SmartLink";
+
+interface IProps {
+    to: string;
+}
+
+export default class AddTheme extends React.Component<IProps> {
+    constructor(props) {
+        super(props);
+    }
+
+    public render() {
+        const classes = currentThemeClasses();
+        return (
+            <React.Fragment>
```

Comment:
Get rid of unused fragment. We we navigating somewhere when we click on the "+"? If not, then this is a button, not a link. To disable clicking in storybook, you can wrap it with a div that has "pointer-events: none"